README.md

```diff
@@ -1 +1,137 @@
-# stencil-progress-bar
+# Stencil Progress Bar
+
+A progress bar that comes in both standard and radial flavours. 
+
+[Link to demo](#)
```

Comment:
We haven't yet determined how demos are going to be hosted, so this feature will have to wait.